docs/en/enterprise-edition/content-collections/runtime-security/agentless-scanning/configure-accounts/configure-aws.adoc

```diff
@@ -19,44 +19,60 @@ Complete the steps below to configure agentless scanning based on the needs of y
 
 . Go to the *Agentless Scanning* section.
 
-. Expand the *Advanced settings*.
+. Expand the *Advanced settings* and provide the appropriate values based on your deployment for the following items.
 +
 image::runtime-security/agentless-aws-configuration.png[width=540]
 
-.. Enable Permissions check to verify that the permissions are correct before running a
-scan.
+.. xref:#aws-agentless-modes[Scan settings]
+.. xref:#aws-agentless-permissions[Scanning permissions]
+.. xref:#aws-agentless-scope[Scanning scope]
+.. xref:#aws-agentless-network[Network resources]
```

Comment:
I'm removing it as requested but it might cause problems for folks following the procedure. That's why I included the links here.

Reply:
Adding headings for each field is an SEO nightmare because we have multiple pages with the exact same headings, it also makes no sense when the section is the size of a single line. I've included headings for the fields where whe have multiple options to make sure the sections have enough content and we can point customers to specific sections of the page as you ask.